code/modules/mob/living/simple_animal/bot/secbot.dm

```diff
@@ -361,11 +361,9 @@
 
 	if(harm)
 		weapon.attack(current_target, src)
-	if(ishuman(current_target))
-		current_target.set_stutter(10 SECONDS)
-		current_target.Paralyze(100)
-		var/mob/living/carbon/human/human_target = current_target
-		threat = human_target.assess_threat(judgement_criteria)
+	if(HAS_TRAIT(current_target, TRAIT_BATON_RESISTANCE) && !current_target.is_staggered())
```

Comment:
Is there a reason as to why you're checking whenever the target is staggered or not if adjust_staggered_up_to is limited to 10 seconds anyways?